boms/legacy-ee/pom.xml

```diff
@@ -1439,6 +1439,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.jboss.resteasy</groupId>
+                <artifactId>resteasy-tracing-api</artifactId>
+                <version>${version.org.jboss.resteasy.extensions}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```

Comment:
We shouldn't add this IMO.

Reply:
@jamezp If I remove above setting, the build problem will occur:

- https://ci.wildfly.org/viewLog.html?buildId=319007&buildTypeId=WF_PullRequest_LinuxJdk11&tab=buildLog

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/201907/182852658-62bf11f4-ccec-46e0-985d-0ab536bbc94e.png">

Reply:
after discussing with James, I guess this can be kept.

Reply:
yes sir!